stock_putaway_product/README.rst

```diff
@@ -0,0 +1,85 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+============================
+Putaway strategy per product
+============================
+
+This module extends the functionality of the odoo putaway strategy.
+It defines a new type of putaway strategy where users can set a specific
+stock location per product.
+
+On the product form, the case, rack, location fields are replaced with a
+specific putaway strategy and location id for the product.
+
+A putaway strategy can be used to ensure that incoming products will be
+stored in the location set on the product form.
+
```

Comment:
I would add:

A recommended set-up is to create a separate Putaway Strategy per Warehouse. This will ensure that the same product will be placed in the appropiate location in each warehouse it is received.